pkg/kudoctl/cmd/init/crds.go

```diff
@@ -189,46 +138,24 @@ func generateOperatorVersion() *apiextv1beta1.CustomResourceDefinition {
 	return crd
 }
 
-// InstanceCrd provides the Instance CRD manifest for printing
+// InstanceCrd provides definition of the instance CRD
 func InstanceCrd() *apiextv1beta1.CustomResourceDefinition {
```

Comment:
Not yours and a nit, but shouldn't this function be private like the others?

Reply:
yep, I actually made that change in another PR so did not want this one to be in conflict with that, but good point 👍 I believe it's the other PR I created yesterday :)